src/index.html

```diff
@@ -11,8 +11,50 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The link to the stylesheet should have an extension of .css, not .scss. SCSS files are preprocessed into CSS files, and the browser can only interpret CSS.